pkg/simulation/config/serviceentry.go

```diff
@@ -0,0 +1,88 @@
+package config
+
+import (
+	networkingv1alpha3 "istio.io/api/networking/v1alpha3"
+	"istio.io/client-go/pkg/apis/networking/v1alpha3"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/howardjohn/pilot-load/pkg/simulation/model"
+)
+
+type ServiceEntrySpec struct {
+	App               string
+	AppType           model.AppType
+	Namespace         string
+	SEResolutionIndex int
+}
+
+type ServiceEntry struct {
+	Spec *ServiceEntrySpec
+}
+
+var _ model.Simulation = &ServiceEntry{}
+
+func NewServiceEntry(s ServiceEntrySpec) *ServiceEntry {
+	return &ServiceEntry{Spec: &s}
+}
+
+func (v *ServiceEntry) Refresh(ctx model.Context) error {
+	v.Spec.SEResolutionIndex = (v.Spec.SEResolutionIndex + 1) % 3
```

Comment:
This is not something often changed, may be more realistic to change something like port.

Or really we should change what we change in Service.go - which appears to be nothing currently

Reply:
Yeah, we can make it similar to service.go as concept is same. Removing Refresh in that case.